common/repositories/base/base_npc_types_repository.h

```diff
@@ -583,6 +604,13 @@ class BaseNpcTypesRepository {
 		e.heroic_strikethrough   = 0;
 		e.faction_amount         = 0;
 		e.keeps_sold_items       = 1;
+		e.chesttexture           = 0;
+		e.combat_hp_regen        = 0;
+		e.combat_mana_regen      = 0;
+		e.aggro_pc               = 0;
+		e.ignore_distance        = 1;
+		e.disable_instance       = 1;
```

Comment:
Are these custom columns?

Reply:
I think this was from a copy of my database when he was testing things at one point.

Reply:
Yes, that is correct @fryguy503 .  I will revert that and make a commit.  This is the cause of all the other comments you just made as well @Kinglykrab  It was due to the recreate of the repo files after I added the is_parcel_merchant with the other db table.

Reply:
Just pushed a commit on this.  Hope I didn't give you a heart attack @Kinglykrab   Should resolve all the other comments as well.